README.md

```diff
@@ -29,6 +25,104 @@ and model-benchmark to measure domain-aware end-to-end deep learning workloads.
 SuperBench is in the early pre-alpha stage for open source, and not ready for general public yet.
 If you want to jump in early, you can try building latest code yourself.
 
+## SuperBench capabilities, workflow and benchmarking metrics
+
+The following graphic shows the capabilities provide by SuperBench core framework and its extension.
+
+<img src="imgs/superbench_structure.png">
+
+Benchmarking metrics provided by SuperBench are listed as below.
+
+<table>
```

Comment:
The table doesn't fully occupy the page.

Reply:
Added a straight line under the headings to keep the equal table width and try to occupy the page.